src/components/Permissions/helpers/permissionsHelper.js

```diff
@@ -106,3 +106,13 @@ export const sortDataByDate = queryResult => {
     (a, b) => new Date(b.created_at) - new Date(a.created_at)
   )
 }
+
+export const filterRemoteRequests = (remoteRequests, slugName) => {
+  const filteredRemoteRequests = { data: [] }
```

Comment:
Maybe you can get rid of the `.data` complexity here and just return a plain array

Reply:
I did it because the function `sortDataByDate` used this format. I changed both function so it's cleaner now